docs/API/LorisRESTAPI.md

```diff
@@ -464,3 +464,235 @@ The format of the JSON object for these URLS is:
+```
+
+To retrieve the session level imaging QC data for a visit, a request can
+be made `/candidates/$CandID/$Visit/qc/imaging`. It will return a JSON object
+of the form
+
+```js
+{
+    "Meta" : {
+        "CandID" : $CandID,
+        "Visit" : $VisitLabel
+    },
+   "QC" : "Pass|Fail"
+   "Pending" : boolean
+}
+```
+
+A PUT to the same location will update the QC information. 
+
+## 4.3 Image Level Data
+```
+GET /candidates/$CandID/$VisitLabel/images/$Filename
+```
+
+Returns raw file with the appropriate MimeType headers for each Filename retrieved from
+`/candidates/$CandID/$Visit/images`.
+
+Only `GET` is currently supported, but future versions of this API may include `PUT`
+support to insert new (or processed) data into LORIS.
+
+## 4.3.1 Image Level QC Data
+```
+GET /candidates/$CandID/$VisitLabel/images/$Filename/qc
+PUT /candidates/$CandID/$VisitLabel/images/$Filename/qc
+```
+
+Returns file level QC information. It will return a JSON object of the form
+
+```js
+{
+    "Meta" : {
+        "CandID" : $CandID,
+        "Visit" : $VisitLabel,
+        "File" : $Filename
+    },
+    "QC" : "Pass|Fail",
+    "Selected" : boolean
+}
+```
+
+`PUT` requests to the same URL will update the QC information.
+
+## 4.4 Alternate formats
+
+There are occasions where you may want to retrieve a file in a different format
+than it is stored in LORIS. This can be achieved by adding `/format/$FormatType`
+to the URL in the API. Currently supported other formats are below. Other formats
+may be added in a future version of this API.
+
+An attempt to convert an image to an unsupported format may result in a
+ `415 Unsupported Media Type` HTTP error.
+
+### 4.4.1 Raw Format
+```
+GET /candidates/$CandID/$VisitLabel/images/$Filename/format/raw
+```
+
+This will return the data in raw format (ie. the output of mnc2raw)
+
+### 4.4.2 BrainBrowser Format
+```
+GET /candidates/$CandID/$VisitLabel/images/$Filename/format/brainbrowser
+```
+
+This (in combination with raw) will let you extract the headers in a JSON
+format that BrainBrowser can load. It will return a JSON object of the format
+
+```js
+{
+    "xspace": {
+        "start":"",
+        "space_length":"",
+        "step":""},
+    "yspace": {
+        "start":"",
+        "space_length":"",
+        "step":""
+    },
+    "zspace": {
+        "start":"",
+        "space_length":"",
+        "step":""
+    },
+    "order":["xspace","zspace","yspace"]
+}
+```
+
+### 4.4.3 Thumbnail Format
+```
+GET /candidates/$CandID/$VisitLabel/images/$Filename/format/thumbnail
+```
+
+This will return a JPEG image that can be used as a thumbnail to represent this
+imaging acquisition statically (such as in the LORIS imaging browser.)
+
+### 4.5 Image Headers
+The LORIS API allows you to extract headers from the images in a RESTful manner.
+The following methods are defined:
+
+### 4.5.1 Header Summary
+```
+GET /candidates/$CandID/$VisitLabel/images/$Filename/headers/
```

Comment:
remove the last slash, as with the last stash it will try and get a specific header